pages/linux/dbclient.md

```diff
@@ -0,0 +1,24 @@
+# dbclient
+
+> Lightweight Dropbear Secure Shell client.
+> More information: <https://manned.org/dbclient.1>.
+
+- Connect to a remote host:
+
+`ssh {{user}}@{{host}}`
```

Comment:
Should this be:
```suggestion
`dbclient {{user}}@{{host}}`
```

Reply:
Oh, why didn't I notice it 😮

Reply:
Oh, yes, of course. You are absolutely right. I am not sure how i managed to do that. I did test the examples with dbclient to ensure they worked as described in the manual, but it seems like i wrote them down too quickly. I was taking some inspiration from the ssh page, and might just have gotten too used to seeing them.

Reply:
No problemo folks. It happens :)
And this is what reviews are for :)
All cool. Thank you for the contribution :)